src/Protocol.cpp

```diff
@@ -396,10 +396,24 @@ Status protocol::parseStatus(uint8_t const* buffer, int size)
     return ret;
 }
 
-template <typename H> H protocol::covarianceNED2NWU(H neu)
+// Transformation that rotates by PI the X axis. Usefull for NED/NWU convertions
+static const Eigen::Matrix3d RotNED2NWU_mat(
+    base::AngleAxisd(M_PI, base::Vector3d::UnitX()));
+static const Eigen::Quaterniond RotNED2NWU_q(
+    base::AngleAxisd(M_PI, base::Vector3d::UnitX()));
+static const Eigen::Matrix3d RotNWU2NED_mat(
+    base::AngleAxisd(-M_PI, base::Vector3d::UnitX()));
+static const Eigen::Quaterniond RotNWU2NED_q(
+    base::AngleAxisd(-M_PI, base::Vector3d::UnitX()));
```

Comment:
is this const needed? Its not used anywhere

Reply:
Removed